package.json

```diff
@@ -71,13 +71,15 @@
     "add-js-extension": "^1.0.4",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^27.4.6",
+    "buffer": "^6.0.3",
```

Comment:
Is this _required_ I'd really rather avoid it if possible?

Reply:
The reason why I added them is that `siwe-parser` depends on `keccak` which depends on `readable-stream` which requires events and buffer but doesn't specify them as dependencies, and esbuild doesn't automatically include browser compatible versions of node modules when bundling.

But it depends on what the goal with the esbuild output is, what is it currently used for? I don't think I know enough about esbuild or npm modules to figure that out from the code, I don't see `dist/esbuild` referenced anywhere in `package.json`. If the bundle is only meant to be used in node environments we could pass `--platform=node` to `esbuild` instead of adding them as dev dependencies. 

Releasing code that works in both node and browser environments is kind of annoying.. I did some more reading and I think that unless we include buffer and events as 'real' dependencies in package.json (I added them as devDependencies) the sdk won't work in browsers unless a bundler is being used that includes those modules by default (like webpack and browserify). And on top of that, I realized that `keccak` relies on a global Buffer (keccak doesn't require/import it) so a browser build will have to provide that as well unless we update the sdk with something like
```ts
import { Buffer } from "buffer";

if (!globalThis.Buffer) globalThis.Buffer = Buffer;
```